src/main/java/com/faforever/client/map/management/MapsManagementController.java

```diff
@@ -0,0 +1,180 @@
+
+  private FilteredList<MapBean> maps = new FilteredList<>(FXCollections.emptyObservableList());
+  private List<String> duplicatedMapNames; // duplicated map names but with different versions
+  private Runnable closeButtonAction;
+
+  @Override
+  public void initialize() {
+    settingChoiceBox();
+    settingTableView();
+    JavaFxUtil.bind(removeButton.disableProperty(), new SimpleListProperty<>(selectedMaps).emptyProperty());
+    updateTableContent();
+  }
+
+  private void updateTableContent() {
+    List<MapBean> clonedList = new ArrayList<>(mapService.getInstalledMaps());
+    List<String> mapNames = clonedList.stream().map(MapBean::getDisplayName).collect(Collectors.toList());
+    this.maps = new FilteredList<>(FXCollections.observableArrayList(clonedList).sorted(), getPredicateByCurrentFilter());
+    this.duplicatedMapNames = clonedList.stream()
+        .filter(map -> map.getVersion() != null && Collections.frequency(mapNames, map.getDisplayName()) > 1)
+        .map(MapBean::getDisplayName).distinct().collect(Collectors.toList());
+    tableView.setItems(this.maps);
+  }
+
+  private void settingChoiceBox() {
+    filterMapsChoiceBox.setConverter(new StringConverter<>() {
+      @Override
+      public String toString(MapFilter object) {
+        return i18n.get(object.getI18n());
+      }
+
+      @Override
+      public MapFilter fromString(String string) {
+        throw new UnsupportedOperationException(); // not required
+      }
+    });
+    filterMapsChoiceBox.getSelectionModel().selectedItemProperty().addListener((obs, oldFilter, newFilter) -> {
+      selectedMaps.clear();
+      maps.setPredicate(getPredicateBy(newFilter));
+      tableView.scrollTo(0);
+    });
+    filterMapsChoiceBox.setItems(FXCollections.observableArrayList(MapFilter.values()));
+    filterMapsChoiceBox.setValue(MapFilter.CUSTOM_MAPS);
+  }
+
+  private Predicate<MapBean> getPredicateByCurrentFilter() {
+    return getPredicateBy(filterMapsChoiceBox.getValue());
+  }
+
+  private Predicate<MapBean> getPredicateBy(MapFilter filter) {
+    return switch (filter) {
+      case OFFICIAL_MAPS -> mapService::isOfficialMap;
+      case CUSTOM_MAPS -> mapService::isCustomMap;
+      case ALL_MAPS -> null;
+    };
+  }
+
+  private void settingTableView() {
+    tableView.setSelectionModel(null);
+    tableView.setRowFactory(param -> new MapBeanRow());
+
+    previewMapColumn.setCellFactory(param -> new PreviewMapCell());
+    previewMapColumn.setCellValueFactory(param -> new SimpleObjectProperty<>
+        (mapService.loadPreview(param.getValue().getFolderName(), PreviewSize.SMALL)));
+
+    mapNameColumn.setCellFactory(param -> new StringCell<>(item -> {
+      String name = item.getDisplayName();
+      ComparableVersion v = item.getVersion();
+      return v != null && duplicatedMapNames.contains(name)
+          ? String.format("%s | %s %s", name, i18n.get("map.version"), v.toString())
+          : name;
```

Comment:
If it is going to be displayed to the user it should be a message and use i18n rather than String.format